test/unit/components/relative.js

```diff
@@ -70,7 +70,7 @@ describe('<FormattedRelative>', () => {
 
         // Should avoid update scheduling tight-loop.
         await sleep(10);
-        expect(setState.calls.length).toBe(1, '`setState()` called unexpectedly');
```

Comment:
Needs to be changed to spy `static getDerivedStateFromProps`.

Reply:
I think this would work (replace `setState` and `spySetState`):
```js
const spyGetDerivedStateFromProps = () => {
  return spyOn(
    require('../../../src/components/relative').BaseFormattedRelative.constructor,
    'getDerivedStateFromProps'
  );
}
```
⚠️ not tested